packages/edit-site/src/components/block-editor/index.js

```diff
@@ -131,6 +135,7 @@ export default function BlockEditor( { setIsInserterOpen } ) {
 
 		return {
 			...restStoredSettings,
+			__unstableFetchMedia: ( _settings ) => fetchMedia( _settings ),
```

Comment:
Why wrap it in an arrow function?

```suggestion
			__unstableFetchMedia: fetchMedia,
```

Reply:
I think I had some more param at first iterations. Thanks!